eva-accession-release/src/main/java/uk/ac/ebi/eva/accession/release/io/VariantContextWriter.java

```diff
@@ -56,10 +64,32 @@ public void open(ExecutionContext executionContext) throws ItemStreamException {
 
         Set<VCFHeaderLine> metaData = new HashSet<>();
         metaData.add(new VCFHeaderLine("reference", referenceAssembly));
-        metaData.add(new VCFInfoHeaderLine("VC", 1, VCFHeaderLineType.String,
+        metaData.add(new VCFInfoHeaderLine(VARIANT_CLASS_KEY, 1, VCFHeaderLineType.String,
                                            "Variant class according to the Sequence Ontology"));
-        metaData.add(new VCFInfoHeaderLine("SID", VCFHeaderLineCount.UNBOUNDED, VCFHeaderLineType.String,
+        metaData.add(new VCFInfoHeaderLine(STUDY_ID_KEY, VCFHeaderLineCount.UNBOUNDED, VCFHeaderLineType.String,
                                            "Identifiers of studies that report a variant"));
+
+        metaData.add(new VCFInfoHeaderLine(VALIDATED_KEY, 0, VCFHeaderLineType.Flag,
+                                           "If an RS was validated by any method according to dbSNP validation "
```

Comment:
Shall we match the phrasing style of other flags? Something like "Validation flag, present when RS was validated by any method as indicated by the dbSNP validation status."